### DISCUSSION TITLE
Why can't some modules change the time discrete parameters？

### DISCUSSION THREAD

**Discussion by Moyiqi** [idaholab/moose#24058] [Q&A General] [answered]
Hello everyone. May I ask a question?
What is the default time discretization in Modules/TensorMechanics/Master，is Newmark or not?
If so, where could I change the parameter α？

Lily

**Comment by GiudGiud** [comment]
Hello

it should show at the beginning of the log, under the mesh information 

Guillaume

**Reply by Moyiqi** [reply]
```
[Modules/TensorMechanics/Master]
    [all]
      # strain = FINITE
      add_variables = true
      generate_output = 'stress_zz strain_zz'
      block = '1 2'
      use_automatic_differentiation = true
    []
  []
  
  [Materials]
    [elasticity]
      type = ADComputeVariableIsotropicElasticityTensor
      poissons_ratio = 0.3
      youngs_modulus = 1e3
      block = '1 2'
    []
    [stress]
      type = ADComputeLinearElasticStress
      block = '1 2'
    []
  []
  
 
  [UserObjects]
    [activated_elem_uo]
      type = ActivateElementsCoupled
      execute_on = timestep_begin
      coupled_var = strain_zz
      activate_value = 0.05
      active_subdomain_id = 1
      expand_boundary_name = 'moving_interface'
    []
  []
  
  [Outputs]
    exodus = true
  []
  [Postprocessors]
    [single_nodal_x]
      type = PointValue
      point = ' 1 1 1'
      variable = "disp_x"
    []
  []
```
When I run this model, which only involves tensor mechanics, modifying scheme parameters has no effect on the results. Whether the equations of tensor mechanics have a default time-discretization scheme, such as using HHT.

Lily

**Reply by GiudGiud** [reply]
How are you changing the time scheme? Could you please paste your executioner block

**Reply by Moyiqi** [reply]
[Executioner]
    type = Transient
  
    automatic_scaling = true
  
    solve_type = 'PJFNK'
  
    line_search = 'none'
    scheme=bdf2
  
    l_max_its = 100
    nl_max_its = 20
    nl_rel_tol = 1e-4
  
    start_time = 0.0
    end_time = 1.0
    dt = 1e-1
    dtmin = 1e-4
  []

Lily

**Reply by hugary1995** [reply, marked as answer]
https://mooseframework.inl.gov/source/timeintegrators/NewmarkBeta.html
